guides/plugins/apps/administration/flow-builder/add-custom-flow-actions-from-app-system.md

```diff
@@ -0,0 +1,255 @@
+
+```text
+└── custom
+    ├── apps
+    │   └── FlowBuilderActionApp
+    │       └── Resources
+    │           └── flow-action.xml
+    │           └── app-icon.png
+    │           └── slack-icon.png
+    │       └── manifest.xml
+    └── plugins
+```
+
+| File name | Description |
+| :--- | :--- |
+| FlowBuilderActionApp | Your app's technical name |
+| app-icon.png | The app's icon |
+| slack-icon.png | Your action icon will be defined for each action in the `flow-action.xml` file. (optional, icons will default to a fallback) |
+| flow-action.xml | Place to define your new actions |
+| manifest.xml | Base information about your app |
+
+### Manifest file
+
+The manifest file is the central point of your app. It defines the interface between your app and the Shopware instance. It provides all the information concerning your app, as seen in the minimal version below:
+
+{% code title="manifest.xml" %}
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
+    <meta>
+        <name>FlowBuilderActionApp</name>
+        <label>Flow Builder Action App</label>
+        <label lang="de-DE">Flow Builder Action App DE</label>
+        <description>This is the example description for app</description>
+        <description lang="de-DE">This is the example description for app</description>
+        <author>shopware AG</author>
+        <copyright>(c) shopware AG</copyright>
+        <version>4.14.0</version>
+        <icon>Resources/app.png</icon>
+        <license>MIT</license>
+    </meta>
+</manifest>
+```
+{% endcode %}
+
+{% hint style="warning" %}
+The name of your app that you provide in the manifest file needs to match the folder name of your app.
+{% endhint %}
+
+## Define the flow action
+
+To create a flow action, you need to define a `<flow-action>` block within a file called `flow-action.xml`. Each `<flow-action>` represents one action and you can define an arbitrary number of actions.
+
+{% code title="Resources/flow-action.xml" %}
+```xml
+<flow-actions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/FlowAction/Schema/flow-action-1.0.xsd">
+    <flow-action>
+        ... # The first action
+    </flow-action>
+    <flow-action>
+        ... # The second action
+    </flow-action>
+    <flow-action>
+        ... # The third action
+    </flow-action>
+    ...
+</flow-actions>
+```
+{% endcode %}
+
+A single flow action would look like this:
+
+```xml
+<flow-action>
+    <meta>
+        <name>slack.send.message</name>
+        <label>Send slack message</label>
+        <label lang="de-DE">Send slack message</label>
```

Comment:
for clarity i would provide the german label here, even if not everybody understands it

same for the DE description